packages/expo-sqlite/ios/EXSQlite.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,268 @@
+// !$*UTF8*$!
```

Comment:
Please remove `packages/expo-sqlite/ios/EXSQlite.xcodeproj`, we no longer add it to unimodules